exercises/wordy/tests/wordy.rs

```diff
@@ -0,0 +1,99 @@
+extern crate wordy;
+
+use wordy::*;
+
+#[test]
+fn addition() {
+    let command = "What is 1 plus 1?";
+    assert_eq!(2, WordProblem::new(command).answer().unwrap());
+}
+
+#[test]
```

Comment:
our other exercises `#[ignore]` all but the first test, should we do so here too?